stations.csv

```diff
@@ -4408,7 +4408,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 5786;Ste-Feyre;ste-feyre;8744500;87445007;46.140102683330724;1.925321817398071;;FR;Europe/Paris;f;f;f;t;f;f;FRXJK;;t;;f;8704670;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv5786;
 5788;St-Jean-de-Luz—Ciboure;st-jean-de-luz-ciboure;8767712;87677120;43.385838;-1.66122;26832;FR;Europe/Paris;f;f;f;t;f;f;FRXJZ;SJZ;t;;f;8700109;f;;f;;f;;f;;f;;f;8700109;f;;f;;f;;;;f;;f;8700192;f;;f;FRXJZ;f;;f;t;;;;Donibane Lohizune;;;;;;;;サン＝ジャン＝ド＝リュズ;;;;Сен-Жан-де-Люз;;;圣让德吕;urn:trainline:public:nloc:csv5788;
 5790;Landerneau;landerneau;8747423;87474239;48.453572;-4.256528;;FR;Europe/Paris;f;f;f;t;f;f;FRXLD;LDO;t;;f;8700453;f;gbsgx5;t;;f;16448;t;;f;;f;8700453;f;;f;;f;;;;f;;f;8700745;f;;f;FRXLD;f;;f;f;;;;;;;;;;;;ランデルノー;;;;;;;朗代尔诺;urn:trainline:public:nloc:csv5790;
-5791;Lens;lens-fr;8734502;87345025;50.426818;2.828254;;FR;Europe/Paris;f;f;f;t;t;f;FRXLE;LNS;t;;f;8700244;t;u0fnbm;t;FRLEN;t;4458;t;;f;;f;;f;;f;;f;;;;f;;f;8700857;f;;f;FRXLE;f;;f;t;;;;;;;;;;;;ランス;;;;;;;;urn:trainline:public:nloc:csv5791;
+5791;Lens;lens-fr;8734502;87345025;50.426818;2.828254;;FR;Europe/Paris;f;f;f;f;t;f;FRXLE;LNS;t;;f;8700244;t;u0fnbm;t;FRLEN;t;4458;t;;f;;f;;f;;f;;f;;;;f;;f;8700857;f;;f;FRXLE;f;;f;t;;;;;;;;;;;;ランス;;;;;;;;urn:trainline:public:nloc:csv5791;
```

Comment:
Disabling the whole station is probably not what you intend to do in this MR. Lens (FR) station is still served by SNCF trains.

Reply:
You're right, I fixed that